src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Apply.cshtml

```diff
@@ -192,7 +192,7 @@
                             <span class="das-task-list__task-tag">
                                 @if (Model.EmailAddresses.Count() > 0)
```

Comment:
Change to  @if (Model.EmailAddresses.Any()) a count will iterate the full collection, any will stop processing at the first entry and drop into the if, more efficient

Reply:
I didn't change this code, so i'll leave it as is - maybe it can be revisited